ui/components/spinner.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function Spinner({ height }) {
+  const borderWidth = Math.ceil(height / 10);
+  const indicatorStyle = { height: `${height}px` };
```

Comment:
It'd be really nice if we could have the dimensions of the spinner flex automatically with its container. Any thoughts about why that might be/not be possible? Definitely a future improvement, though.